### PR TITLE
chore(forge): kB to B in build --sizes

### LINKED ISSUE #7567: [Improvement] change forge build sizes report to Bytes
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (dbc48ea 2024-04-03T00:17:03.697365187Z)

### What command(s) is the bug in?

forge build

### Operating System

Linux

### Describe the bug

`forge build --sizes` lists values for size and margin in kB. It is unclear whether 1000x or 1024x

It would be more clear if listed in Bytes right aligned as the sizes are up to 5 digits anyway.

image

To become : 

```
Compiler run successful!
| Contract                       | Size (B)  | Margin (B)  |
|--------------------------------|-----------|-------------|
| Address                        |        86 |      24 490 |
| SomeHugeContract               |    17 906 |         667 |
```

### PR DESCRIPTION
**Pull request by kamuik16**
Closes #7567

### CODE REVIEW

**Comment by kamuik16** [comment]
Example Output:
```bash
Compiler run successful!
| Contract | Size (B) | Margin (B) |
|----------|----------|------------|
| Counter  | 247      | 24329      |
```

File: crates/common/src/compile.rs [hunk 1]
```diff
@@ -419,8 +419,8 @@ impl Display for SizeReport {
 
             table.add_row([
                 Cell::new(name).fg(color),
-                Cell::new(contract.size as f64 / 1000.0).fg(color),
-                Cell::new(margin as f64 / 1000.0).fg(color),
+                Cell::new(contract.size as f64).fg(color),
+                Cell::new(margin as f64).fg(color),
```
Comment (onbjerg) [inline crates/common/src/compile.rs:423]: we can remove the `as f64` casts

**Review by onbjerg — CHANGES_REQUESTED** [review]

**Review by Evalir — APPROVED** [review]
lgtm

**Review by DaniPopes — APPROVED** [review]
Thanks!

**Pull request merged**